views/aboutus/main.js

```diff
@@ -1,58 +0,0 @@
-// Make sure the settings are fetched.
```

Comment:
So if you look at the about us page, it is still currently being set to white even in light mode. Where exactly is this being stored?